reflectometry/amor_reduction_detailed.ipynb

```diff
@@ -348,12 +349,35 @@
    "source": [
     "max_w = sc.max(data.events.coords['wavelength'])\n",
     "min_w = sc.min(data.events.coords['wavelength'])\n",
-    "bins = sc.Variable(values=np.linspace(min_w.value, max_w.value, 100), \n",
-    "                   unit=data.events.coords['wavelength'].unit,\n",
-    "                   dims=['wavelength'])\n",
+    "bins = sc.linspace(\n",
+    "    dim='wavelength', \n",
+    "    start=min_w.value, \n",
+    "    stop=max_w.value, \n",
+    "    num=100,\n",
+    "    unit=data.events.coords['wavelength'].unit)\n",
     "sc.histogram(data.events, bins).plot()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "7fb6a9fc",
+   "metadata": {},
+   "source": [
+    "In addition to plotting a histogram of the events as a function of wavelength, we can visualise a subset in a table."
```

Comment:
Actually you can visualize all: You get a slider if you omit the `['event', :10]`.

Reply:
Is it possible to have the number of rows spinbox default to a smaller value than 50? Something like 

`sc.table(data['event', :], nrows=10)`

Reply:
it is actually using a setting from the config:
https://github.com/scipp/scipp/blob/main/python/src/scipp/table.py#L292
but it's true that we could make it a function argument.

Reply:
See https://github.com/scipp/scipp/issues/2034.